common/web/gesture-recognizer/src/test/resources/json/canaryRecording.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
Note that it is ill-advisable to spend much time reviewing the recorded-sequence .json files found in this folder.

I've pretty-printed them so that they may be easily read and edited manually if necessary... but all of this is largely auto-generated.  These files account for all but about 850 lines of changes at the time of this comment.